torch/_refs/__init__.py

```diff
@@ -6128,6 +6128,184 @@ def vdot(self, other):
 log_normal_ = _make_inplace(log_normal)
 zero_ = _make_inplace(zero)
 
+
+def _isStorage(obj):
+    # TODO: this is slightly inaccurate; we don't allow subclasses on
+    # UntypedStorage
```

Comment:
why not type check directly?

Reply:
Hmm, so, originally I was Dynamo'ing through this code and Dynamo didn't support the type() is ... test. But then I realized I didn't need to Dynamo through it I could make_fx trace it instead, probably can fix it now